crates/sui-graphql-e2e-tests/tests/objects/staked_sui.move

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# init --simulator --accounts C
+
+//# run-graphql
+{ # Initial query yields only the validator's stake
```

Comment:
suggestion: you can also put the comment before all the curly braces so it doesnt get wiped out if you use graphiql to construct these queries

Reply:
I don't really understand this comment (I don't use GraphiQL to construct these queries, so maybe that's why) -- how does the comment get wiped out if you put it at this position?

I'll leave it here for now, just to not hold up landing this stack, but can follow-up with a clean-up once I understand!

Reply:
if you have comments in a .graphql snippet, and paste it into graphiql, prettifying it will wipe out all comments. but  it's not an issue if you don't use graphiql to format your snippets